zebra-test/src/lib.rs

```diff
@@ -24,6 +25,14 @@ pub mod prelude;
 pub mod transcript;
 pub mod vectors;
 
+/// A Tokio runtime that can be shared between tests.
+pub static RUNTIME: Lazy<tokio::runtime::Runtime> = Lazy::new(|| {
```

Comment:
Docs: Can you explain when developers should use the shared runtime, and when they should create their own runtime?

Can you also mention that the runtime is multi-threaded?

Reply:
Done in [218db63](https://github.com/jvff/zebra/commit/218db638c8000944045fae6216ec8a17d827360a).